nautobot/core/tests/test_graphql.py

```diff
@@ -1006,22 +1006,20 @@ def test_query_circuit_termination_connected_endpoint(self):
     @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
     def test_query_config_context_and_custom_field_data(self):
 
-        query = """
-        query {
-            devices {
+        query = f"""
```

Comment:
To my mind, converting the graphql queries to f-strings is a loss of readability. I'd prefer to see them as-is with an appropriate `pylint: disable` comment.